epoxy-processor/src/main/java/com/airbnb/epoxy/processor/GeneratedModelWriter.kt

```diff
@@ -484,12 +493,13 @@ class GeneratedModelWriter(
         return methods
     }
 
-    private fun getLayoutDimensions(modelInfo: GeneratedModelInfo): Pair<CodeBlock, CodeBlock> {
+    private fun getLayoutDimensions(modelInfo: GeneratedModelInfo): Pair<CodeBlock?, CodeBlock?> {
```

Comment:
use a nullable Pair instead of making the contents nullable
```suggestion
    private fun getLayoutDimensions(modelInfo: GeneratedModelInfo): Pair<CodeBlock, CodeBlock>? {
```